config/bootstrap.rb

```diff
@@ -32,3 +32,10 @@ def perform
     add_transaction_tracer :perform, :category => :task
   end
 end
+
+require 'routemaster/mixins/log'
+include Routemaster::Mixins::Log
```

Comment:
You're adding to the global namespace — not ideal?

Instead of forcing this in the bootstrap, you could for instance choose the path of lazyness and `warn` on the first log call.